RELEASE.md

```diff
@@ -0,0 +1,74 @@
+# Versioning and releasing Google Api Compiler
+This document states how we plan to version and release the Google Api Compiler.
+1. How and when to release Google Api Compiler
+2. How to setup development branch for day to day commits.
+
+## Branches
+This Api Compiler repo contains only _one_ branch:
+
+`master` : This has the lastest code into which all the contribution
```

Comment:
"lastest" -> "latest"

Also, since you use "PR" later, how about defining that acronym...

 This has the latest code into which all the contribution PRs (pull requests) go...